### PR TITLE
added different maven command and memory for kie-wb-common FDB

### PR DESCRIPTION
**Pull request by mbiarnes**
only for kie-wb-common FDB:
raise the JVM to 7g and disable the multithreading

### CODE REVIEW

File: job-dsls/jobs/downstream_pr_jobs.groovy [hunk 1]
```diff
@@ -168,17 +168,25 @@ for (repoConfig in REPO_CONFIGS) {
...
+                if (repo != "kie-wb-common") {
+                    project / 'builders' << 'hudson.tasks.Maven' {
+                        mavenName("kie-maven-${Constants.MAVEN_VERSION}")
+                        jvmOptions("-Xms1g -Xmx4g -XX:+CMSClassUnloadingEnabled")
+                        targets("-e -fae -nsu -B -T1C clean install -s \$SETTINGS_XML_FILE -Dkie.maven.settings.custom=\$SETTINGS_XML_FILE -Dfull -DskipTests")
+                    }
+                } else {
+                    project / 'builders' << 'hudson.tasks.Maven' {
+                        mavenName("kie-maven-${Constants.MAVEN_VERSION}")
+                        jvmOptions("-Xms1g -Xmx7g -XX:+CMSClassUnloadingEnabled")
```
Comment (mareknovotny) [inline job-dsls/jobs/downstream_pr_jobs.groovy]: I see in https://github.com/kiegroup/kie-wb-common/blob/master/kie-wb-common-stunner/kie-wb-common-stunner-showcase/kie-wb-common-stunner-showcase-project/pom.xml#L1297 there is another try out to have even bigger heap size value limit. So is this expected to be there @romartin ?

Comment (romartin) [reply job-dsls/jobs/downstream_pr_jobs.groovy]: yeah @mareknovotny @mbiarnes - it's possible to 'override' the settings given in the maven `gwt` plugin's configuration, by using jvm system properties. You can check all those in the [compile goal's reference](https://gwt-maven-plugin.github.io/gwt-maven-plugin/compile-mojo.html).

So for example, the `extraJvmArgs` can be changed from command line by using `-Dgwt.extraJvmArgs='...'`

**Comment by romartin** [comment]
Hey guys, not approving yet as the issues are still not solved yet, let's keep in touch. Or do you need this merged for testing too?

**Comment by mbiarnes** [comment]
@romartin @winklerm @mareknovotny 
https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/pullrequest/job/kie-wb-common-downstream-pullrequests/ changed online
with -T1C added and JVM Options= -Xms1g -Xmx16g -XX:+CMSClassUnloadingEnabled
Lets see how the kie-wb-common-downstream-pullrequests behaves. If it works properly this PR will have to be modified.

**Comment by mbiarnes** [comment]
@mareknovotny @romartin the kie-wb-common FDB are configured the same online - so these PR won't change nothing - bt it has to be merged since not the older version of FDB will override and change the mem size and re-activate the multithreading. This PR has to be merged since it blocks other pRs in Jenkins scripts to be merged.

**Comment by mareknovotny** [comment]
how ended the setup with 11g? @mbiarnes

**Comment by mbiarnes** [comment]
@mareknovotny via ssh it wworked properly. But it was not tested via Jenkins job. 16g workes for sure on Jenkins.

**Pull request merged**